central/views/common/types.go

```diff
@@ -1,9 +1,15 @@
 package common
 
-// ResourceCountByCVESeverity is the count of resources affected by cve distributed over severity.
+// ResourceCountByCVESeverity provides functionality to retrieve the count of resources affected by cve distributed over severity.
 type ResourceCountByCVESeverity interface {
-	GetCriticalSeverityCount() int
-	GetImportantSeverityCount() int
-	GetModerateSeverityCount() int
-	GetLowSeverityCount() int
+	GetCriticalSeverityCount() ResourceCountByFixability
+	GetImportantSeverityCount() ResourceCountByFixability
+	GetModerateSeverityCount() ResourceCountByFixability
+	GetLowSeverityCount() ResourceCountByFixability
+}
+
+// ResourceCountByFixability provides functionality to retrieve the count of resources affected by cve distributed over fixable property.
+type ResourceCountByFixability interface {
```

Comment:
In a first go, it is confusing for me to grasp the fact that these counts depend on what the "Resource" is. Considering global scope for simplicity, if the "Resource" is "CVE", then fixable counts fixable CVEs in the system and total counts all CVEs in the system. But when the "Resource" is something else like Image, fixability counts images where a given CVE is fixable, and total counts images where a given CVE exists. I get this after reading it a few times. Maybe it is because of hangover from how these terms were used in the legacy graphQL implementation.

Would adding more explanation/examples in comments make the distinctions more clear?